tests/Integration/Database/QueryBuilderTest.php

```diff
@@ -203,6 +203,24 @@ public function testOrWhereTime()
         $this->assertSame(2, DB::table('posts')->where('id', 1)->orWhereTime('created_at', new Carbon('2018-01-02 03:04:05'))->count());
     }
 
+    public function testWhereIn()
+    {
+        $this->assertEquals('select * from "posts" where "id" in (?, ?)',
+            DB::table('posts')->whereIn('id', [1, 2])->toSql());
+
+        $this->assertEquals('select * from "posts" where ("id", "title") in ((?, ?), (?, ?))',
+            DB::table('posts')->whereIn(['id', 'title'], [[1, 'Foo Post'], [2, 'No Post']])->toSql());
+    }
+
+    public function testWhereNotIn()
+    {
+        $this->assertEquals('select * from "posts" where "id" not in (?, ?)',
+            DB::table('posts')->whereNotIn('id', [1, 2])->toSql());
+
+        $this->assertEquals('select * from "posts" where ("id", "title") not in ((?, ?), (?, ?))',
+            DB::table('posts')->whereNotIn(['id', 'title'], [[1, 'Foo Post'], [2, 'No Post']])->toSql());
```

Comment:
I would add a test with empty arrays just to check that the default `1 = 0` is there.